packages/eslint-config-commons-ui/index.js

```diff
@@ -93,7 +93,12 @@ module.exports = [
           "newlines-between": "always",
         },
       ],
-
+      "import/no-unresolved": [
+        "error",
+        {
+          ignore: ["@payloadcms/richtext-lexical/react"],
```

Comment:
Does this throw error even in eslint? I thought it was just jest.

Reply:
This also happens with eslint but this update on [jsconfig](https://github.com/CodeForAfrica/ui/pull/1160/commits/3c1f80a5017bc6cdce3641700a54334791e64a6d#diff-32bb4ec8a706f55a4996ffa013329d15ccc1a6aef5871d2d7667477d55eb901e) seems to fix.